src/resolvers/space.ts

```diff
@@ -12,12 +13,13 @@ export default async function resolve(key) {
         method: 'post',
         data: {
           query: `query { space(id: "${key}") { avatar } }`
-        }
+        },
+        ...axiosDefaultParams
```

Comment:
I think we should pass `axiosDefaultParams` in `fetchHttpImage` itself, is there any reason to import it here on all resolvers? 🤔

Reply:
These params are injected in the graphql axios query, and not fetchImate

Reply:
ahh makes sense. i thought its the same